src/pages/Locations/LocationDetails/LocationDetails.jsx

```diff
@@ -0,0 +1,40 @@
+import {useEffect, useState} from 'react';
+import {useLocation} from "react-router-dom";
+import useLocationPhotos from "../../../hook/useLocationPhotos";
+import './LocationDetails.css';
+import LocationTitle from "./components/LocationTitle/LocationTitle";
+import LocationPhotosGrid from "./components/LocationPhotosGrid/LocationPhotosGrid";
+
+
+const LocationDetails = () => {
+    const {state} = useLocation();
+    const {location, locationName, locationId} = state;
+    const locationRegion = location.region ? location.region + ", " : ''
+    const locationCountry = location.country ?? ''
```

Comment:
Codacy has a fix for the issue: [Replace `····const·locationCountry·=·location.country·??·''` with `··const·locationCountry·=·location.country·??·"";`](https://app.codacy.com/gh/mohammadnedaei/Nearby-Locations-React/pullRequest?prid=9855911)
```suggestion
  const locationCountry = location.country ?? "";
```